docs/SETUP_REMOTE.md

```diff
@@ -1,12 +1,15 @@
 # Remote Machine Setup
 
-## Dependencies
+## Required Dependencies
```

Comment:
We don’t have optional dependencies, so from my point of view it is quite obvious that dependencies are required.